### ISSUE TITLE
Lighten background colour of substances index (AA)

### ISSUE THREAD

**Issue by ainemckay** [MHRA/products#920]
## 1.4.3 Contrast (minimum) (AA) – Small text

Issue: Blue (#2472B6) link text on light blue (#E6EAF4)

## Recommendation
Colour contrast should meet the minimum colour contrast ratio 4.5:1. Lighten the background light blue to (#F1F3F9).

## Acceptance criteria
- [x] Background to search index should be #F1F3F9

## Testing instructions
- Go to non-prod site
- Inspect the search index box and check the background colour is #F1F3F9
